src/components/Playlists/PlaylistVideoCard.js

```diff
@@ -0,0 +1,83 @@
+import { Card, Text, Button, Icon } from '../Reusable'
+import { usePlaylists, useTheme, useWatchlater } from '../../contexts'
+import { getTextColor, getBgColor, getIconColor } from '../../utils'
+
+const PlaylistVideoCard = ({ video: {
+    _id,
+    url,
+    title,
+    description,
+    views,
+    uploadedOn,
+    category
+}, playlistId, watchlaterVideo
+}) => {
+    const { theme } = useTheme()
+    const { removeVideoFromPlaylist, showPlaylistsAlert, playlistsDispatch } = usePlaylists()
+    const { removeFromWatchlater, watchlaterDispatch } = useWatchlater()
+
+    async function handleRemoveVideoFromWatchlater() {
+        const removeFromWatchlaterResponse = await removeFromWatchlater(_id)
+        if (!(removeFromWatchlaterResponse === 404 || removeFromWatchlaterResponse === 500)) {
```

Comment:
Yes, i have used the try catch in `removeFromWatchlater` in context file. I have used `!` in if condition so it doesn't execute in case of any error. But I'll improve the code so that it doesn't look confusing.